templates/web/bromley/footer.html

```diff
@@ -9,20 +9,7 @@
 
         <div class="nav-wrapper">
             <div class="nav-wrapper-2">
-                <div id="main-nav" role="navigation">
-                    <ul class="nav-menu nav-menu--main">
-                        <li><[% IF c.req.uri.path == '/' %]span[% ELSE %]a href="/"[% END
```

Comment:
This one lacks the class all(?) the others have, but now it will have it - this makes a difference to display on e.g. /faq, so it either needs resetting in CSS or not including.